src/runtime/components/WysiwygContent.vue

```diff
@@ -1,59 +1,56 @@
 <template>
   <div :class="[wrapperDefaultClasses, wrapperClasses]">
-    <component :is="tag" :class="[defaultClasses, classes]" v-html="htmlContent"></component>
+    <article :class="[defaultClasses, classes]" v-html="htmlContent"></article>
```

Comment:
why is this an article tag ?

Reply:
This is because the old dynamic component tags can lead to invalid or improperly nested HTML like nesting  headings or anchor links (i.e.: an h1 inside an h2). And I chose article because it is the recommended tag wrapper for dynamic htmls that wont cause any invalid html